src/index.css

```
@@ -38,6 +38,17 @@ body,
     transform: translateY(0);
   }
 }
+/* Pushed screens (Sheet variant="screen") enter from the right, the way a
+   forward navigation reads — as opposed to sheetUp, which says "temporary
+   surface over the page". */
+@keyframes screenIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "screenIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)